app/views/user/manage.phtml

```diff
@@ -62,6 +62,7 @@
 		<thead>
 			<tr>
 				<th><?= _t('admin.user.username') ?></th>
+				<th><?= _t('admin.user.language') ?></th>
```

Comment:
I'm not sure it's useful to have it in the list, we should pick only the most useful information to avoid a too big table. What do you think?

Reply:
I don't mind removing it. I feel it could be useful in the list unlike other information. I agree that only useful information should be displayed in the list.

Reply:
I guess it depends on the use case ^^